scripts/dumps/lib/serialize_entity_in_turtle.coffee

```diff
@@ -23,6 +38,13 @@ module.exports = (entity)->
     else
       console.warn yellow('missing formatter'), datatype
 
+  text += """\n  wikibase:statements #{statementsCount} ;"""
```

Comment:
maybe `"""\n  wikibase:statements count #{statementsCount} ;"""` ? to be clear about what is this integer exactly

Reply:
I don't get it, what is the role of that `count` keyword in the triple?